JobConfig/digitize/OffSpill.fcl

```diff
@@ -10,8 +10,11 @@ outputs.TriggeredOutput.SelectEvents : [
 # Lower thresholds for LoopHelix triggerable
 physics.filters.Triggerable.MinParticleMom : 50.0
 physics.filters.Triggerable.MaxParticleMom : 300.0
-physics.trigger_paths : [ "0:DigitizePath", "1:TriggerablePath" , "2:TriggerableCHPath" , @sequence::Trig_physMenu.trigger_paths]
-outputs.TriggerableOutput.SelectEvents : [ "TriggerablePath", "TriggerableCHPath" ]
+# Define the default list of triggerable paths
+physics.TriggerablePaths : [ "TriggerablePath", "TriggerableCHPath"]
+# Define the trigger path
+physics.trigger_paths : [ "0:DigitizePath", @sequence::physics.TriggerablePaths,  @sequence::Trig_physMenu.trigger_paths]
```

Comment:
the 0:DigitizePath index is redundant and somewhat confusing now given TriggerablePaths are no longer explicitly indexed. Please remove the index.

Reply:
I removed this index for now, and we can add this back if we are able to update to have bits in the SelectEvents in the future.